src/rgw/CMakeLists.txt

```diff
@@ -283,10 +284,12 @@ target_include_directories(rgw_common
   PUBLIC "${LUA_INCLUDE_DIR}")
 
 if(WITH_RADOSGW_D4N)
-  add_dependencies(rgw_common cpp_redis)
+  add_dependencies(rgw_common cpp_redis) # remove later -Sam
   target_link_libraries(rgw_common PRIVATE cpp_redis)
   target_include_directories(rgw_common SYSTEM PUBLIC "${CMAKE_SOURCE_DIR}/src/cpp_redis/includes")
   target_include_directories(rgw_common SYSTEM PUBLIC "${CMAKE_SOURCE_DIR}/src/cpp_redis/tacopie/includes")
+
+  target_include_directories(rgw_common SYSTEM PUBLIC "${CMAKE_SOURCE_DIR}/src/boost_redis/include")
```

Comment:
i had to add `OpenSSL::SSL` to the `target_link_libraries()` to build with `WITH_RADOSGW_AMQP_ENDPOINT=OFF`. i assume it's because `boost/redis/connection.hpp` is pulling in `boost/asio/ssl.hpp`